constants/Bingo.json

```
@@ -1,347 +1,351 @@
+      "difficulty": "Easy",
       "note": [
-        "Complete F2 Dungeon Runs"
+        "Gain coins from the Scavenger Enchant"
       ],
       "guide": [
-        "This needs Combat 15 and a completion",
-        "of the 2 previous floors to enter.",
-        "To efficiently grind this, you should",
-        "get help from Party Finder/Friends."
+        "Apply/Use the highest tier of Scavenger",
+        "that you can get your hands on and put it",
+        "on your main weapon. Gifts from the Jerry Island",
+        "can be a way to obtain Scavenger IV books."
       ]
     },
-    "Strange Effects": {
+    "Yummers": {
       "difficulty": "Easy",
       "note": [
-        "Kill a Runic Mob"
+        "15 Cans of Worms"
       ],
       "guide": [
-        "Look for mobs with a purple name",
-        "and kill it."
+        "Those have a 20% drop chance from Trash Gobblers",
+        "in the Backwater Bayou (11.3% base catch chance)",
+        "and require Fishing Level 5+ to fish up.",
+        "Use the best Fishing Gear you have access to."
       ]
     },
-    "Meiser": {
-      "difficulty": "Easy",
+    "Exterminator": {
+      "difficulty": "Medium",
       "note": [
-        "50k coins in the bank"
+        "Vacuum 100 Pests"
       ],
       "guide": [
-        "You will get this from leveling up",
-        "skills for the Skill Average goal.",
-        "Remember to actually put the money",
-        "you get in the Bank :)"
+        "Pest spawns have a cooldown of 5 minutes while online",
+        "and an hour while offline. While actively farming, there's",
+        "a 0.2% chance of a pest spawn each broken mature crop.",
+        "Bonus Pest Chance can increase the amount of spawned pests",
+        "at once while actively farming. You can also use up to 3",
+        "Pest Traps (purchased from Philip in the Garden) to get Pests",
+        "while doing other goals. Pests unlock from Garden Level 5.",
+        "Use the Vacuum Philip hands you when talking to him."
       ]
     },
-    "Reinforced Reforger": {
-      "difficulty": "Easy",
+    "Spider Squisher": {
+      "difficulty": "Medium",
       "note": [
-        "Apply the Rare Diamond Reforge Stone"
+        "75 Spiders in 90s"
       ],
       "guide": [
-        "Mine Diamonds in the Deep Caverns until",
-        "you get a Rare Diamond to apply.",
-        "A safe spot to do this is between the Diamond Reserve",
-        "and the Obsidian Sanctuary (-20 27 70).",
-        "Mobs don't spawn there."
+        "Use a Super Cleaver from Ophelia in the Dungeon Hub",
+        "(80k coins + Cleaver to buy)",
+        "or a Dreadlord Sword from Dungeons to achieve this.",
+        "Make sure to apply Cleave 5 on your Cleaver.",
+        "§cSpiders on your Private Island don't count!"
       ]
     },
-    "Ender Pearl Collector": {
+    "The Mirror Dimension": {
       "difficulty": "Hard",
       "note": [
-        "45k Ender Pearl Collection"
+        "1.000s Rift Time"
```

```suggestion
        "1,000s Rift Time"
```
Let's use the English separator.